test/client/data/live-node-list/getBodyByTagName.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Title</title>
+        <script src="/hammerhead.js" class="script-hammerhead-shadow-ui"></script>
+        <script>
+            function bodyExists () {
+                return document.getElementsByTagName('body').length === 1;
+            }
+
+            const hammerhead = window['%hammerhead%'];
+
+            hammerhead.get('./utils/destination-location').forceLocation('http://localhost/sessionId/http://origin_iframe_host');
+            hammerhead.start({ crossDomainProxyPort: 2000 });
+
+            window.top.bodyExistsBeforeAttachingBody = bodyExists();
```

Comment:
I think that it is unnecessary to save the test data in the top window. Just save it in the current iframe and use `iframe.contentWindow.<prop>` in your test.